src/mongocxx/options/modify_collection.cpp

```diff
@@ -35,7 +35,8 @@ void modify_collection::no_padding(bool no_padding) {
 void modify_collection::index(bsoncxx::document::view_or_value index_spec,
                               std::chrono::seconds seconds) {
     _index.emplace(document{} << "keyPattern" << bsoncxx::types::b_document{std::move(index_spec)}
-                              << "expireAfterSeconds" << seconds.count() << finalize);
+                              << "expireAfterSeconds" << bsoncxx::types::b_int64{seconds.count()}
```

Comment:
clang-format wrapped it that way.  I had the `finalize` on the same line, originally.